addons/report/views/layouts.xml

```diff
@@ -117,6 +117,7 @@
             </t>
 
             <ul class="list-inline">
+                <li t-if="o and o.name"><span t-field="o.name"/></li>
```

Comment:
I tried but the runbot crashes with None-related issues.
I'll revert to the initial test.